controllers/microservices_controller.go

```diff
@@ -31,16 +39,72 @@ type MicroservicesReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.2/pkg/reconcile
 func (r *MicroservicesReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	log := log.FromContext(ctx)
+	log = log.WithValues("tempo", req.NamespacedName)
+	tempo := v1alpha1.Microservices{}
+	if err := r.Get(ctx, req.NamespacedName, &tempo); err != nil {
+		if !apierrors.IsNotFound(err) {
+			log.Error(err, "unable to fetch TempoMicroservices")
+		}
+
+		// we'll ignore not-found errors, since they can't be fixed by an immediate
+		// requeue (we'll need to wait for a new notification), and we can get them
+		// on deleted requests.
+		return ctrl.Result{}, client.IgnoreNotFound(err)
+	}
+
+	objects, err := manifests.BuildAll(manifests.Params{Tempo: tempo})
+	// TODO(pavolloffay) check error type and change return appropriately
+	if err != nil {
+		return ctrl.Result{
+			Requeue:      true,
+			RequeueAfter: time.Second,
+		}, err
+	}
+	errCount := 0
```

Comment:
Is `errCount` used? Looks like its increased in case of failure. But never used?

Reply:
good catch, the count should be checked

Reply:
did you add a check?

Reply:
yes

Reply:
Pushed the change now